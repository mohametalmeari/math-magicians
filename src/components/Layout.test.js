import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Layout from './Layout';

describe('Layout Component', () => {
  test('test render of Layout', () => {
    const { container } = render(
      <MemoryRouter>
        <Layout />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
