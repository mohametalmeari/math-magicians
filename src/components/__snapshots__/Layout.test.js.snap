// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component test render of Layout 1`] = `
<div>
  <div
    class="wrapper"
  >
    <nav>
      <h1>
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          Math Magicians
        </a>
      </h1>
      <ul
        class="menu"
      >
        <li
          class="page-link"
        >
          <a
            aria-current="page"
            class="active"
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class="page-link"
        >
          <a
            class=""
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li
          class="page-link"
        >
          <a
            class=""
            href="/quote"
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
